__tests__/decorators.spec.ts

```diff
@@ -66,4 +66,177 @@ describe('Decorators', () => {
       expect(consoleSpy).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('arraySort', function() {
+    
```

Comment:
DRY, you can use a factory function to generate all the test classes:
```
function testClass(array, config) => {
 return new class {
    @SortBy('name', config)
    testingArray = array
 }
}
```
Create a `describe` for each type you support and test both ascending and descending.
Also, test negative numbers and a `0` in the number type.

Reply:
Love it as well, also addressed. New spec file looks much cleaner :)